app/models/theatre.py

```diff
@@ -0,0 +1,47 @@
+"""PC est magique Flask App - Rooms Models"""
+
+from __future__ import annotations
+
+import datetime
+import typing
+
+import sqlalchemy as sa
+
+from app import db
+from app.utils.columns import (
+    column,
+    one_to_many,
+    many_to_one,
+    Column,
+    Relationship,
+)
+
+
+Model = typing.cast(type[type], db.Model)  # type checking hack
+
+
+class Spectacle(db.Model):
+    """All information about ESPCI theatre spectacle information."""
+
+    id: Column[int] = column(sa.Integer(), primary_key=True)
+    name: Column[str] = column(sa.String(120), nullable=False)
+    description: Column[str] = column(sa.String(2000), nullable=True)
+    ticket_link: Column[str] = column(sa.String(120), nullable=True)
```

Comment:
```suggestion
    description: Column[str | None] = column(sa.String(2000), nullable=True)
    ticket_link: Column[str | None] = column(sa.String(120), nullable=True)
```

ou alors, le rendre non nullable avec un `default=""`, comme tu préfères